src/base-element.js

```diff
@@ -134,6 +135,14 @@ export class BaseElement {
     this.resources_ = resourcesFor(this.getWin());
   }
 
+  /**
+  * Returns the element priority. A value from 0 to 10.
```

Comment:
should I put the explanation of this function in base-element.js or custom-element.js or both?